db/migrate/20200117172135_add_trigger_to_sync_post_replies.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class AddTriggerToSyncPostReplies < ActiveRecord::Migration[6.0]
+  def up
+    # we don't want this column to be readonly yet
+    Migration::ColumnDropper.drop_readonly(:post_replies, :reply_id)
```

Comment:
Is this safe in the situation where it was never marked as readonly?

Reply:
Yes, it's executing `DROP FUNCTION IF EXISTS`